src/components/AddPeopleAndRoles/PeopleAndRoles.vue

```diff
@@ -110,12 +110,14 @@ export default class PeopleAndRoles extends Mixins(EntityFilterMixin) {
   @Action setAddPeopleAndRoleStepValidity!: ActionBindingIF
 
   private newOrgPerson: OrgPersonIF = {
-    id: null,
-    firstName: '',
-    lastName: '',
-    middleName: '',
-    orgName: '',
-    type: '',
+    person: {
+      id: null,
+      firstName: '',
+      lastName: '',
+      middleName: '',
+      orgName: '',
+      partyType: ''
```

Comment:
If you change this property to an enum type, you may need to initialize it to null.